scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -120,47 +129,56 @@ workflow GvsCalculatePrecisionAndSensitivity {
 
 task IsVcfOnChromosomes {
   input {
-    File input_vcf
+    File input_vcf_fofn
+    Int index
     Array[String] chromosomes
   }
 
-  String output_vcf_name = basename(input_vcf)
   command <<<
-    set -o errexit -o nounset -o pipefail
+    # Prepend date, time and pwd to xtrace log entries.
+    PS4='\D{+%F %T} \w $ '
+    set -o errexit -o nounset -o pipefail -o xtrace
+
+    # Get a one-based line number from the zero-based input index.
+    line_number=$((~{index} + 1))
+    input_vcf_path=$(sed -n "${line_number}p" ~{input_vcf_fofn})
 
-    cat ~{input_vcf} | gunzip | grep -v '^#' | cut -f 1 | sort | uniq > chrom.txt
+    input_vcf=$(basename ${input_vcf_path})
+    gcloud storage cp ${input_vcf_path} ${input_vcf}
+    cat ${input_vcf} | gunzip | grep -v '^#' | cut -f 1 | sort | uniq > chrom.txt
     NL=$(cat chrom.txt | wc -l)
     if [ $NL -ne 1 ]; then
-      echo "~{input_vcf} has either no records or records on multiple chromosomes"
+      echo "${input_vcf_path} has either no records or records on multiple chromosomes"
       exit 1
     fi
 
     mkdir output
-    touch output/~{output_vcf_name}
+    output_vcf_name=${input_vcf}
+    touch output/${output_vcf_name}
     VCF_CHR=$(cat chrom.txt)
     chromosomes=( ~{sep=' ' chromosomes} )
 
     for i in "${chromosomes[@]}"
     do
       if [ $VCF_CHR = $i ]; then
-        cp ~{input_vcf} output/~{output_vcf_name}
-        echo "Including ~{input_vcf} as it is on $i."
+        cp ${input_vcf} output/${output_vcf_name}
+        echo "Including ${input_vcf_path} as it is on $i."
         break
       else
-        touch output/~{output_vcf_name}
-        echo "Skipping ~{input_vcf} as it is not on $i."
+        touch output/${output_vcf_name}
+        echo "Skipping ${input_vcf_path} as it is not on $i."
       fi
     done
   >>>
 
   runtime {
-    docker: "gcr.io/gcp-runtimes/ubuntu_16_0_4:latest"
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-07-24-alpine-bd6b9d62e"
```

Comment:
Do we need to use our docker?  Wouldn't gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-alpine be smaller and also work?

Reply:
Oh probably so; I needed to step up from `ubuntu` to get `gcloud`, but I don't think I need all the other stuff in our image.